stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/ChatInfoViewModel.kt

```diff
@@ -30,6 +31,13 @@ class ChatInfoViewModel(
     init {
         _state.value = State()
         viewModelScope.launch {
+            val channelControllerResult = chatDomain.useCases.getChannelController(cid).await()
+            if (channelControllerResult.isSuccess) {
+                val canDeleteChannel =
+                    channelControllerResult.data().members.value?.firstOrNull { it.getUserId() == chatDomain.currentUser.id }?.isOwnerOrAdmin
```

Comment:
Would it be possible to have `canDeleteChannel()` logic in one function and reuse in `ChannelActionsViewModel`?

Reply:
I'll create `List<Member>::isCurrentUserOwnerOrAdmin` extension function